docs/api/server/Environment-Variables.md

```diff
@@ -36,34 +36,39 @@ One App can be configured via Environment Variables:
   * [`HOLOCRON_SERVER_MAX_SIM_MODULES_FETCH`](#holocron_server_max_sim_modules_fetch)
   * [`ONE_ENABLE_POST_TO_MODULE_ROUTES`](#one_enable_post_to_module_routes)
   * [`ONE_MAP_POLLING_MAX`](#one_map_polling_max)
+  * [`ONE_MAP_POLLING_MIN`](#one_map_polling_min)
   * [`ONE_REFERRER_POLICY_OVERRIDE`](#one_referrer_policy_override)
   * [`ONE_SERVICE_WORKER`](#one_service_worker)
 
-**Alphabetical Contents**
-* [`HOLOCRON_MODULE_MAP_URL`](#holocron_module_map_url)
-* [`HOLOCRON_SERVER_MAX_MODULES_RETRY`](#holocron_server_max_modules_retry)
-* [`HOLOCRON_SERVER_MAX_SIM_MODULES_FETCH`](#holocron_server_max_sim_modules_fetch)
-* [`HTTPS_PORT`](#https_port)
-* [`HTTPS_PRIVATE_KEY_PASS_FILE_PATH`](#https_private_key_pass_file_path)
-* [`HTTPS_PRIVATE_KEY_PATH`](#https_private_key_path)
-* [`HTTPS_PUBLIC_CERT_CHAIN_PATH`](#https_public_cert_chain_path)
-* [`HTTPS_TRUSTED_CA_PATH`](#https_trusted_ca_path)
-* [`HTTP_METRICS_PORT`](#http_metrics_port)
-* [`HTTP_ONE_APP_DEV_CDN_PORT`](#http_one_app_dev_cdn_port)
-* [`HTTP_ONE_APP_DEV_PROXY_SERVER_PORT`](#http_one_app_dev_proxy_server_port)
-* [`HTTP_PORT`](#http_port)
-* [`IP_ADDRESS`](#ip_address)
-* [`NODE_ENV`](#node_env)
-* [`ONE_CLIENT_CDN_URL`](#one_client_cdn_url)
-* [`ONE_CLIENT_LOCALE_FILENAME`](#one_client_locale_filename)
-* [`ONE_CLIENT_REPORTING_URL`](#one_client_reporting_url)
-* [`ONE_CLIENT_ROOT_MODULE_NAME`](#one_client_root_module_name)
-* [`ONE_CONFIG_ENV`](#one_config_env)
-* [`ONE_ENABLE_POST_TO_MODULE_ROUTES`](#one_enable_post_to_module_routes)
-* [`ONE_MAP_POLLING_MAX`](#one_map_polling_max)
-* [`ONE_MAP_POLLING_MIN`](#one_map_polling_min)
-* [`ONE_REFERRER_POLICY_OVERRIDE`](#one_referrer_policy_override)
-* [`ONE_SERVICE_WORKER`](#one_service_worker)
+<details>
+  <summary>Alphabetical Contents</summary>
+
+  * [`HOLOCRON_MODULE_MAP_URL`](#holocron_module_map_url) ⚠️
+  * [`HOLOCRON_SERVER_MAX_MODULES_RETRY`](#holocron_server_max_modules_retry)
+  * [`HOLOCRON_SERVER_MAX_SIM_MODULES_FETCH`](#holocron_server_max_sim_modules_fetch)
+  * [`HTTPS_PORT`](#https_port)
+  * [`HTTPS_PRIVATE_KEY_PASS_FILE_PATH`](#https_private_key_pass_file_path)
+  * [`HTTPS_PRIVATE_KEY_PATH`](#https_private_key_path)
+  * [`HTTPS_PUBLIC_CERT_CHAIN_PATH`](#https_public_cert_chain_path)
+  * [`HTTPS_TRUSTED_CA_PATH`](#https_trusted_ca_path)
+  * [`HTTP_METRICS_PORT`](#http_metrics_port)
+  * [`HTTP_ONE_APP_DEV_CDN_PORT`](#http_one_app_dev_cdn_port)
+  * [`HTTP_ONE_APP_DEV_PROXY_SERVER_PORT`](#http_one_app_dev_proxy_server_port)
+  * [`HTTP_PORT`](#http_port)
+  * [`IP_ADDRESS`](#ip_address)
+  * [`NODE_ENV`](#node_env)
+  * [`ONE_CLIENT_CDN_URL`](#one_client_cdn_url)
+  * [`ONE_CLIENT_CSP_REPORTING_URL`](#one_client_csp_reporting_url) ⚠️
+  * [`ONE_CLIENT_LOCALE_FILENAME`](#one_client_locale_filename)
+  * [`ONE_CLIENT_REPORTING_URL`](#one_client_reporting_url) ⚠️
+  * [`ONE_CLIENT_ROOT_MODULE_NAME`](#one_client_root_module_name) ⚠️
+  * [`ONE_CONFIG_ENV`](#one_config_env)
```

Comment:
```suggestion
  * [`ONE_CONFIG_ENV`](#one_config_env) ⚠️
```
Hmm `ONE_CONFIG_ENV` is required when using `appConfig.provideStateConfig` I believe... wondering if we should just include a note instead of labeling it here.

Reply:
Hey @Francois-Esquire ! Please let me know how you'd like me to update this. I can "commit suggestion" for both of these, or amend the commit with these changes. I think we'll also need to add `⚠️` to the regular table of contents in addition to the alphabetical one. Thanks!

Reply:
@pcooney10 you can commit the suggestions, just ensure you change the commit message from the default one to follow commitlint, but if you are adding stuff that is not in the suggestion you might as well add them manually from your machine

Reply:
@infoxicator thank you for responding! @pcooney10 Yes, as suggested, if you think it would be better to update manually, please feel free to do so!

Reply:
Sweet - should be all set! Thanks!